### ISSUE TITLE
Add better automation for upgrades

### ISSUE THREAD

**Issue by dylanmccall** [flathub/com.spacestation14.Launcher#1]
In general, updating this repo to use a newer version of the project is a simple process:

- In [com.spacestation14.Launcher.yaml](https://github.com/flathub/com.spacestation14.Launcher/blob/master/com.spacestation14.Launcher.yaml), update the git source for `space-station-14-launcher` to refer to the newest tag from https://github.com/space-wizards/SS14.Launcher/releases.
- (If we need a new version of dotnet, _probably_ just `s/dotnetX/dotnetY/g` and try not to panic).
- Run tools/update-ss14-sources.py to update the nuget sources files.
- Update appdata.xml. (Pulling a description from a github release would be nice)
- Test the new build.

So, pretty much the whole thing can be automated. It would be nice if we add a tool which automatically checks for and applies updates, creating a pull request so it is easy to verify and release the update all in one place.

**Comment by PJB3005** [comment]
Yeah, my current workflow for updating the existing launcher builds is as follows:
0. (update the file on our server that forces updates for manual downloads)
1. Push new tag to GitHub (Actions publishes build)
2. Merge master into Steam private repo, push tag there too (Actions publishes build)
3. Set new build live on Steam

Being able to automate this repo would be nice as part of step 1.
